python/uperf/test_uperf_plugin.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import socket
+import unittest
+import uperf_plugin
+from arcaflow_plugin_sdk import plugin
+
+
+class ExamplePluginTest(unittest.TestCase):
+    @staticmethod
+    def test_serialization():
+        plugin.test_object_serialization(
+            uperf_plugin.UPerfParams("127.0.0.1",
+                uperf_plugin.IProtocol.TCP, 5
+            )
+        )
+
+        plugin.test_object_serialization(
+            uperf_plugin.UPerfServerParams(5)
+        )
+
+        plugin.test_object_serialization(
+            uperf_plugin.UPerfResults("test", {})
+        )
+
+        plugin.test_object_serialization(
+            uperf_plugin.UPerfServerResults()
+        )
+
+        plugin.test_object_serialization(
+            uperf_plugin.UPerfError("Some error")
+        )
+
+        plugin.test_object_serialization(
+            uperf_plugin.UPerfServerError(1, "Not a real error")
+        )
+
+    def test_functional(self):
+        # Test the server succesfully exiting
+        server_1sec_input = uperf_plugin.UPerfServerParams(1)
+
+        output_id, _ = uperf_plugin.run_uperf_server(server_1sec_input)
+
+        self.assertEqual("success", output_id)
+        # The output data is currently empty.
+
+        # --------------------------
+        # Test error when port is in use
+        # Make socket for port be in use.
```

Comment:
grammatical error?

Reply:
I don't think so. The first one is active tense, and the second one is passive tense.

Reply:
Clarified comment.